integration-tests/deployment/ccip/view/v1_0/rmn_proxy_contract.go

```diff
@@ -0,0 +1,33 @@
+package v1_0
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment/ccip/view/types"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/rmn_proxy_contract"
+)
+
+type RMNProxyView struct {
+	types.ContractMetaData
+	ARM common.Address `json:"arm"`
```

Comment:
what do you think about making it RMN instead? I understand the method is GetARM but we have named this as RMNProxy